zebra/zapi_msg.c

```diff
@@ -1348,8 +1348,6 @@ static void zread_fec_register(ZAPI_HANDLER_ARGS)
 
 	s = msg;
 	zvrf = vrf_info_lookup(VRF_DEFAULT);
-	if (!zvrf)
-		return;
```

Comment:
Agreed with Mark, this will introduce code coverage issues .
I have seen the following code coverage bug when accessing a pointer without a null check . "Null pointers should not be dereferenced".

I feel, having this check is always safe.

Reply:
I don't know if coverage is smart enough, but some places (like `fpm_lsp_send()`) don't have these null check.